chef/data_bags/crowbar/migrate/network/011_add_ovs_attributes.rb

```diff
@@ -0,0 +1,43 @@
+def upgrade(ta, td, a, d)
+  a["network"]["nova_floating"]["add_ovs_bridge"] = ta["network"]["nova_floating"]["add_ovs_bridge"]
+  a["network"]["nova_floating"]["bridge_name"] = ta["network"]["nova_floating"]["bridge_name"]
+  a["network"]["nova_fixed"]["add_ovs_bridge"] = ta["network"]["nova_fixed"]["add_ovs_bridge"]
+  a["network"]["nova_fixed"]["bridge_name"] = ta["network"]["nova_fixed"]["bridge_name"]
+
+  # We need to switch 'add_ovs_bridge' to true on nodes that have the
+  # "neutron-network" role and use the 'openvswitch' mechanism driver
+  nodes = NodeObject.find("roles:neutron-network")
+  nodes.each do |node|
```

Comment:
Actually it might be better to move this part of the migration to the neutron barclamp. If only to avoid the duplication with https://github.com/crowbar/crowbar-openstack/pull/23/files#diff-1cab7f3b6842279ae4d6bd875e073cacR324